kevm-pyk/src/kevm_pyk/kevm.py

```diff
@@ -177,7 +172,6 @@ def concrete_rules() -> List[str]:
             'EVM-TYPES.#asByteStack',
             'EVM-TYPES.#asByteStackAux.recursive',
             'EVM-TYPES.#asWord.recursive',
-            'EVM-TYPES.ByteArray.range',
             'EVM-TYPES.bytesRange',
```

Comment:
Instead of deleting, should it be updated in the same way that `concrete-rules.txt` is updated?

Reply:
@ehildenb This list already contains an `'EVM-TYPES.#range'`. I assumed I would duplicate it by renaming it.

Reply:
I called out the other list I'm talking about with a comment, have a look. Basically, the concrete_rules list in `kevm.py` and the `concrete-rules.txt` file should agree with each other.